src/renderTopLanguages.js

```diff
@@ -72,6 +94,32 @@ const renderTopLanguages = (topLangs, options = {}) => {
   if (hide_title) {
     height -= 30;
   }
+
+  if(layout === 'compact') {
+    let offset = 25
+    height -= 120;
+
+    return `
+      <svg width="${width+50}" height="${height}" viewBox="0 0 ${width+50} ${height}" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <style>
+          .header { font: 600 18px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${titleColor} }
+          .lang-name { font: 400 14px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${textColor} }
+        </style>
+        <rect data-testid="card-bg" x="0.5" y="0.5" width="99.7%" height="99%" rx="4.5" fill="${bgColor}" stroke="#E4E2E2" />
+        ${hide_title ? '' : `<text data-testid="header" x="25" y="35" class='header'>Top Languages</text>`}
```

Comment:
As i said on the previous review comment, this whole logic is unneeded and duplicate

Reply:
I tried to render conditionally but its taking time because I have a very small amount of SVG knowledge. I'll spend a few hours rendering them conditionally. 

Can I use nested SVG for positioning?

Reply:
Yes you can nest svgs inside svg. Try it

Reply:
Hopefully, I resolved your last comment regarding conditionally rendering. 😊